src/config/dark-sites.config

```diff
@@ -161,6 +162,7 @@ deepweblinks.net
 del.dog
 deltarune.com
 demonforums.net
+ders.eba.gov.tr/ders/proxy/*/liveMiddleware/eba.html
```

Comment:
I really have no idea what this is, but probably won't work due to `*` in the middle of the string.

Reply:
I can send a screenshot of it here but I have to remove it anyway because `.../proxy/VERSION/...` changes frequently except if you have something that place-holds any value (if this feature doesn't exist I would recommend making it).

Reply:
>  (if this feature doesn't exist I would recommend making it).

#2517  :)

Reply:
> but probably won't work due to * in the middle of the string.

If #373 is resolved then `ders.eba.gov.tr/ders/proxy/*/liveMiddleware/eba.html` should work!
Here's a screenshot of it, should I add it as it was before? if not, just merge it :)
![2021-02-26 09_50_58-Window](https://user-images.githubusercontent.com/61620817/109411649-a7ba2300-79b4-11eb-8d00-24a2361bfc7d.png)